java-frontend/src/test/java/org/sonar/java/SonarComponentsTest.java

```diff
@@ -454,14 +455,15 @@ public void feedback_should_not_be_sent_in_sonarLintContext_or_when_collecting_i
 
   }
 
+  @Ignore
   @Test
   public void ucfg_activation_should_rely_on_active_rules() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Fix or remove this skipped unit test ([squid:S1607](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.java%3Ajava&pullRequest=1966&issues=AWJtT8OJsgPCf6tZNy0n&open=AWJtT8OJsgPCf6tZNy0n)

[ISSUE_KEY]: AWJtT8OJsgPCf6tZNy0n